### PR TITLE
Add Trustworthy Mod

### PR DESCRIPTION
**Pull request by PillTime**
Added a mod that disables feign notes.
I had never touched C# before, so I checked https://github.com/taulazer/tau to see how to write mods.
I set the score to 80% because the mod makes the beatmaps easier, but I'm not sure if that's the best multiplier.

### CODE REVIEW

File: osu.Game.Rulesets.Hishigata/Mods/HishigataModTrustworthy.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+namespace osu.Game.Rulesets.Hishigata.Mods
+{
+    public class HishigataModTrustworthy : Mod, IApplicableToBeatmapConverter
+    {
+        public override string Name => "Trustworthy";
+        public override string Acronym => "TW";
+        public override LocalisableString Description => HishigataModTrustworthyStrings.ModDescription;
+        public override double ScoreMultiplier => 0.8;
+        public override ModType Type => ModType.Conversion;
+        //public override bool UserPlayable => true;
```
Comment (LumpBloom7) [inline osu.Game.Rulesets.Hishigata/Mods/HishigataModTrustworthy.cs:17]: Remove these comments

File: osu.Game.Rulesets.Hishigata/Mods/HishigataModTrustworthy.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        public override IconUsage? Icon => FontAwesome.Solid.Check;
+        //public override bool HasImplementation => true;
```
Comment (LumpBloom7) [inline osu.Game.Rulesets.Hishigata/Mods/HishigataModTrustworthy.cs:19]: As above

**Review by LumpBloom7 — CHANGES_REQUESTED** [review]
Looks good overall. Some leftover comments need to be removed.

In addition, a resx file needs to be created in resource for the strings used in TW, since that is where the localised strings are fetched from (also makes the string visible in crowdin).

**Comment by PillTime** [comment]
I copied the resx file that was already there for the hidden mod and changed the value, not sure if anything else should be done.

**Review by LumpBloom7 — APPROVED** [review]
LGTM. Everything seems to be in order.

**Pull request merged**